Examples/Algorithms/Fatras/src/NuclearInteractionOptions.cpp

```diff
@@ -0,0 +1,233 @@
+// This file is part of the Acts project.
+//
+// Copyright (C) 2021 CERN for the benefit of the Acts project
+//
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+#include "ActsExamples/Fatras/NuclearInteractionOptions.hpp"
+
+#include "ActsExamples/Utilities/Options.hpp"
+
+#include <string>
+
+#include <TFile.h>
+#include <TH1F.h>
+#include <TVectorF.h>
```

Comment:
Is there a chance to move the ROOT dependency into the ROOT folder?

Reply:
Moved